src/Sarif.UnitTests/TestData/SarifVersionOneToCurrentVisitor/ExpectedOutputs/OneRunWithAllReportingDescriptors.sarif

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2019-02-20",
+  "version": "2.0.0-csd.2.beta.2019-02-20",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "CodeScanner",
+          "semanticVersion": "2.1.0",
+          "ruleDescriptors": [
+            {
+              "id": "C2003",
+              "name": "Rule C2003",
+              "shortDescription": {
+                "text": "Rules were meant to be broken."
+              },
+              "fullDescription": {
+                "text": "This is a test for validating modification of tool.driver.ruleDescriptors"
```

Comment:
>tool.driver.ruleDescriptors [](start = 71, length = 27)

i might need a bit of help in coming up with more TCs to ensure all paths are tested. Will sync up offline. I am ok adding those as part of separate PR is required.